dartboard/DartboardView.py

```diff
@@ -0,0 +1,299 @@
+        self.radius = 0
+        self.scores = ["6", "10", "15", "2", "17", "3", "19", "7", "16", "8", "11", "14", "9", "12", "5", "20", "1", "18", "4", "13"]
+        self.score_labels = []
+
+        #this list holds the points
+        self.points = []
+
+        #names window
+        self.setWindowTitle("Pyside2 Drawing Polygon")
+
+        #sets initial geometry for the window
+        self.setGeometry(self.WindowPosX, self.WindowPosY, self.WindowSizeX, self.WindowSizeY)
+
+        #draws the circles and dots for the first time
+        self.DrawRegions()
+
+        #shows the window
+        self.show()
+
+
+    #this function is called when a point is placed on the window
+    def mousePressEvent(self, event):
+
+         #this call could be used to get your mock points        
+         #print(event.pos())
+         #
+
+         # this call could be used to get your mock points
+         print(event.pos())
+
+         #
+         hit = DartboardHit(x=event.pos().x(), y=event.pos().y())
+         hit.save()
+         print(DartboardHit.objects.all())
+
+         #this stores the real position of the click by adding together the top left position of the window with the click position relative to the window
+         realPosition = QPoint(event.pos().x() + self.WindowPosX, event.pos().y() + self.WindowPosY)
+
+         #check for collisions with each region
+         if (self.bullseye.containsPoint(realPosition, Qt.OddEvenFill)):
+             print("bullseye")
+         #elif (self.outer_bullseye.containsPoint(realPosition, Qt.OddEvenFill)):
+             #print("outer bullseye")
+
+         #this line adds an entry for the point that was just placed. It calculates the x and y distance from the center of the circle divided by the current radius of the circle
+         #this is used to relocate the points later when the window is resized
+         self.points.append([(event.pos().x() - (self.WindowSizeX / 2.0)) / self.radius, (event.pos().y() - (self.WindowSizeY / 2.0)) / self.radius])
+
+         #this line draws the point on the screen where the mouse was clicked
+         self.scene.addEllipse(self.WindowPosX + event.pos().x(), self.WindowPosY + event.pos().y(), 3,3, QPen(), QBrush(Qt.white))
+
+    #this function is called when the window is resized
+    def resizeEvent(self, event):
+        #print(self.sceneRect())
+
+        #these lines set the variables initialized in the constructor to their new values
+        self.WindowSizeX = event.size().width()
+        self.WindowSizeY = event.size().height()
+        self.WindowPosX = self.pos().x()
+        self.WindowPosY = self.pos().y()
+
+        #this line sets the Scene's geometry based on the newly resized window
+        self.scene.setSceneRect(self.WindowPosX, self.WindowPosY, self.WindowSizeX, self.WindowSizeY)
+
+        #recall the drawing function to replace the circles and points with the new size
+        self.DrawRegions()
+
+        #tbh idek what this does
```

Comment:
I'm aware of what the super class does.. I just didn't see any effect on the code when this call was commented out or not.

Reply:
Oh, I see, I think we leave it then. It could be doing something important on the backend that isn't readily visible to us right now. I think we still should remove the comment.